### PR TITLE
Openshift cleanup + secure data loading

### PR DESCRIPTION
**Pull request by GregSutcliffe**
As per https://github.com/theforeman/foreman-infra/pull/340/#discussion_r144057218 I've kept the crons, but in a more sane place. Likewise the post_deploy script may come in useful so I've kept that. The rest of the first commit is cleanup.

Second commit goes with https://github.com/theforeman/foreman-infra/pull/346. I've tested this in production using an unused variable name (secret_token2) and echoed it to stdout, seems to load from the secure file just fine.

### CODE REVIEW

File: Gemfile.lock.ruby-2.0 [hunk 1]
```diff
@@ -1,6 +1,5 @@
 GEM
   remote: https://rubygems.org/
-  remote: https://rubygems.org/
   remote: http://rubygems.org/
```
Comment (ekohl) [inline Gemfile.lock.ruby-2.0]: Is this still needed?

Comment (GregSutcliffe) [reply Gemfile.lock.ruby-2.0]: Not sure, I've just copied whats on disk right now. I'd be inclined to leave it alone for the moment and investigate with a copy of the repo....

**Comment by GregSutcliffe** [comment]
Ty @ekohl

**Comment by GregSutcliffe** [comment]
Email sending still working, all good.

**Pull request merged**